src/wallet/rpcdump.cpp

```diff
@@ -694,6 +708,15 @@ UniValue dumpwallet(const JSONRPCRequest& request)
         }
     }
     file << "\n";
+    for (const CScriptID &scriptid : scripts) {
+        CScript script;
+        std::string address = EncodeDestination(scriptid);
+        if(pwallet->GetCScript(scriptid, script)) {
+            file << strprintf("%s 0 script=1", HexStr(script.begin(), script.end()));
+            file << strprintf(" # addr=%s\n", address);
```

Comment:
We don't test the comment?

Reply:
What do you mean?

Reply:
Nevermind, `address` is parsed in
https://github.com/bitcoin/bitcoin/blob/4db82b7aab4ad64717f742a7318e3dc6811b41be/test/functional/wallet-dump.py#L27-L33